content/document.ts

```diff
@@ -46,7 +46,7 @@ const getHTMLPath = (folder: string) => path.join(folder, HTML_FILENAME);
 const getMarkdownPath = (folder: string) =>
   path.join(folder, MARKDOWN_FILENAME);
 
-export function updateWikiHistory(
+export async function updateWikiHistory(
```

Comment:
Also needs to be `await`ed here:

https://github.com/mdn/yari/blob/5a9af2b3781562425ca57149a76cb0eb8563d457/tool/sync-translated-content.ts#L209